packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -896,14 +898,56 @@ const Conversation = WebexPlugin.extend({
         async: false
       });
     }
+    /**
+   * helper function to cloneDeepWith for copying instance function
+   * @param {T} value
+   * @returns {null}
+   */
+    // eslint-disable-next-line consistent-return
+    const customActivityCopy = (value) => {
+      const {files} = params.body.object;
```

Comment:
Why are you using the externally scoped `params` object here? Isn't it being passed in as `value` via your `cloneDeepWith` function?

Reply:
Complete params is not pass every time in value. Value is a property that need to copy from params, it is a recursive operation over params but here i am adding condition to check if value match SCRObject then do copy of few more function that generally avoided. @adamweeks  Do you have better approach in mind to do same ?  Please suggest.